editor/script_editor_debugger.cpp

```diff
@@ -1620,7 +1620,7 @@ ScriptEditorDebugger::ScriptEditorDebugger(EditorNode *p_editor) {
 		reason->set_text("");
 		reason->set_editable(false);
 		hbc->add_child(reason);
-		reason->add_color_override("font_color", Color(1, 0.4, 0.0, 0.8));
+		reason->add_color_override("font_color", Color(0.1, 0.9, 0.8, 0.9));
```

Comment:
what about `p_editor->get_gui_base()->get_color("highlight_color", "Editor");` ?
of course this does not has to be green.
It would fit with the rest of the theme. and the theme color, since it is present everywhere, also is not alarming.
In addition to that green and blue are really popular highlight colors anyway.